.github/workflows/build.yml

```diff
@@ -117,6 +117,48 @@ jobs:
         name: wheels
         path: dist
 
+  build-linux-s390x:
+    runs-on: ubuntu-latest
+    needs: [lint]
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: ['s390x']
+    steps:
+    - name: Building package on ${{ matrix.platform }}
+      uses: appleboy/ssh-action@v0.1.10
+      env:
+        GH_REPOSITORY: ${{ github.server_url }}/${{ github.repository }}
+      with:
+        host: ${{secrets.S390X_SSH_HOST}}
+        username: ${{secrets.S390X_SSH_USER }}
+        key: ${{secrets.S390X_SSH_KEY}}
```

Comment:
The use of secrets will break building any PR, including this one.

Reply:
If those secret variables are included , then it won't break at all. I already allowed edits and access to secrets to maintainers